kvmagent/kvmagent/test/test_fence_vm_on_suspect_host.py

```
@@ -0,0 +1,296 @@
+        ssh_argv = (
+            "timeout %d sshpass -f %s ssh -p %d "
+            "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null "
+            "-o ConnectTimeout=5 -o ServerAliveInterval=5 -o ServerAliveCountMax=2 "
+            "-o BatchMode=no %s@%s %s"
+        ) % (ssh_timeout, sshpass_file, target_port, target_user, target_ip,
+             linux_mod.shellquote(remote_cmd))
+
+        s = shell_cls(ssh_argv)
+        try:
+            s(False)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)